web/src/stream_popover.js

```diff
@@ -5,6 +5,7 @@ import assert from "minimalistic-assert";
 import render_inline_decorated_stream_name from "../templates/inline_decorated_stream_name.hbs";
 import render_move_topic_to_stream from "../templates/move_topic_to_stream.hbs";
 import render_left_sidebar_stream_actions_popover from "../templates/popovers/left_sidebar/left_sidebar_stream_actions_popover.hbs";
+import render_stream_info_popover from "../templates/popovers/stream_info_popover.hbs";
 
 import * as blueslip from "./blueslip";
 import * as browser_history from "./browser_history";
```

Comment:
I suggest creating a separate file `stream_card_popover.ts` to not mix with `stream_popover`